### PR TITLE
feat: Add fee information to swap status [WEB-482]

### PR DESCRIPTION
**Pull request by niklasnatter**

### CODE REVIEW

File: packages/shared/src/consts.ts [hunk 1]
```diff
@@ -3,7 +3,10 @@ import { ChainflipNetwork, ChainflipNetworks } from './enums';
 const POOLS_NETWORK_FEE_HUNDREDTH_PIPS: Partial<
   Record<ChainflipNetwork, number>
 > = {
-  [ChainflipNetworks.perseverance]: 100,
+  [ChainflipNetworks.backspin]: 1000,
```
Comment (niklasnatter) [inline packages/shared/src/consts.ts:6]: unfortunately this is not available on any rpc yet

File: packages/swap/src/routes/__tests__/fee.test.ts
Comment (niklasnatter) [inline packages/swap/src/routes/__tests__/fee.test.ts:1]: this was never used. as we are reading the fee info from the rpc directly now we can remove it

File: packages/swap/src/event-handlers/swapExecuted.ts [hunk 2]
```diff
@@ -15,16 +16,34 @@ export default async function swapExecuted({
...
     event.args,
   );
+  const swap = await prisma.swap.findUnique({
+    where: { nativeId: swapId },
+  });
+
+  // skip update if we are not tracking swap
+  if (!swap) {
+    return;
+  }
```
Comment (acdibble) [inline packages/swap/src/event-handlers/swapExecuted.ts:29]: we should be tracking all swaps now

Comment (niklasnatter) [reply packages/swap/src/event-handlers/swapExecuted.ts:29]: i think so, right? i was scared about breaking something but will remove the check

File: packages/swap/src/routes/swap.ts [hunk 3]
```diff
@@ -156,6 +165,7 @@ router.get(
       egressAmount: swap?.egress?.amount?.toString(),
       egressScheduledAt: swap?.egress?.scheduledAt?.valueOf(),
       egressScheduledBlockIndex: swap?.egress?.scheduledBlockIndex,
+      paidFess: swap?.fees,
```
Comment (niklasnatter) [inline packages/swap/src/routes/swap.ts:168]: add test and type

**Review by acdibble — APPROVED** [review]

File: packages/shared/src/consts.ts (hunk 1 above)
Comment (niklasnatter) [reply packages/shared/src/consts.ts:7]: https://linear.app/chainflip/issue/WEB-683/use-network-fee-from-rpc-instead-of-hardcoding-it

File: packages/sdk/src/swap/types.ts [hunk 4]
```diff
@@ -108,6 +108,7 @@ export type SwapStatusResponse = CommonStatusFields &
         egressAmount: string;
         egressScheduledAt: number;
         egressScheduledBlockIndex: string;
+        paidFess: SwapFee[];
```
Comment (acdibble) [inline packages/sdk/src/swap/types.ts:111]:
```suggestion
        feesPaid: SwapFee[];
```

**Pull request merged**